odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/AlertRepositoryImpl.java

```diff
@@ -138,6 +206,21 @@ public long countByOwner(final long ownerId) {
             .orElse(0L);
     }
 
+    @Override
+    public long countDependentObjectsAlerts(final long ownerId) {
+        final List<String> ownOddrns = getObjectsOddrnsByOwner(ownerId);
+
+        final CommonTableExpression<Record1<String>> cte = getChildOddrnsLinageByOwnOddrnsCte(ownOddrns);
+
+        return dslContext.with(cte)
+            .select(countDistinct(ALERT.ID))
+            .from(ALERT)
+            .join(cte.getName()).on(field(cte.getName() + ".child_oddrn", String.class).eq(ALERT.DATA_ENTITY_ODDRN))
```

Comment:
the same as above

Reply:
https://github.com/opendatadiscovery/odd-platform/pull/281#discussion_r761895681